lib/helpers/tags.js

```diff
@@ -160,12 +165,17 @@ function getLinksTag(req) {
 }
 
 /**
+ * Renders Mustache tags with req data, if tags are included in the given string.
+ *
  * @param {String} string
  * @param {Object} req
  * @return {String}
  */
 async function render(string, req) {
-  return mustache.render(string, await module.exports.getTags(req));
+  // Check for opening and closing double brackets.
+  const hasTags = /\{{.*?\}}/g.exec(string);
```

Comment:
Could this use the one-or-more `+?`
```suggestion
  const hasTags = /\{{.+?\}}/g.exec(string);
```

So we don't match on blank tags? (edge case-y - yes).